adserver/management/commands/rtdimport.py

```diff
@@ -296,6 +296,27 @@ def import_advertisements(self, advertisements_data):
             "small",
         ]
 
+        with open(
```

Comment:
I assume this was automated? We can probably just delete this file.

Reply:
I agree this file can be deleted. We're never running it again. I'll do a separate PR after merging this.